src/util/irep_ids.def

```diff
@@ -665,6 +665,7 @@ IREP_ID_TWO(C_no_initialization_required, #no_initialization_required)
 IREP_ID_TWO(overlay_class, java::com.diffblue.OverlayClassImplementation)
 IREP_ID_TWO(overlay_method, java::com.diffblue.OverlayMethodImplementation)
 IREP_ID_ONE(annotations)
+IREP_ID_TWO(cprover_rounding_mode, __CPROVER_rounding_mode)
```

Comment:
Please don't put any hardcoded `__CPROVER` in here. See above for how to do it instead.

Reply:
'll do this, but can you please explain why? Is it just to make the CPROVER_PREFIX configurable?

Reply:
Yes. You might argue that there is a lot more that needs cleanup in this regard, but let's at least not introduce it in even more places. (I think we've got an issue open to do this cleanup.)